connectors/kinesis-sink/3.0/kinesis-sink.md

```diff
@@ -0,0 +1,127 @@
+language: Java
+document:
+license: Apache License 2.0
+tags: ["Pulsar IO", "Kinesis", "Sink"]
+alias: Kinesis Sink
+features: ["Use Kinesis sink connector to sync data from Pulsar"]
+license_link: "https://www.apache.org/licenses/LICENSE-2.0"
+icon: "/images/pulsar-hub.svg"
+support: StreamNative
+support_link: https://streamnative.io
+support_img: "/images/streamnative.png"
+owner_name: ""
+owner_img: ""
+dockerfile: https://hub.docker.com/r/streamnative/pulsar-io-kinesis
+id: "kinesis-sink"
+source: "https://github.com/streamnative/pulsar/tree/v3.0.0.1/pulsar-io/kinesis"
+download: "https://github.com/streamnative/pulsar/releases/download/v3.0.0.1/pulsar-io-kinesis-3.0.0.1.nar"
+---
+
+The AWS Kinesis sink connector pulls data from Pulsar and persists data into Amazon Kinesis. For more information about connectors, see [Connector Overview](https://docs.streamnative.io/docs/connector-overview).
+
+![](/images/connectors/kinesis-sink.png)
+
+This document introduces how to get started with creating an AWS Kinesis sink connector and get it up and running.
+## Quick start
+
+### Prerequisites
+
+The prerequisites for connecting an AWS Kinesis sink connector to external systems include:
+
+1. Create a Kinesis data stream in AWS.
+2. Create an [AWS User](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html) and an `AccessKey`(Please record the value of `AccessKey` and its `SecretKey`).
+3. Assign the following permissions to the AWS User:
+- [AmazonKinesisFullAccess](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonKinesisFullAccess.html) 
+- [CloudWatch:PutMetricData](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricData.html): it is required because AWS Kinesis producer will periodically [send metrics to CloudWatch](https://docs.aws.amazon.com/streams/latest/dev/monitoring-with-cloudwatch.html).
+
+### 1. Create a connector
+Depending on the environment, there are several ways to create an AWS Kinesis sink connector:
+
+- [Create a Connector on StreamNative Cloud](https://docs.streamnative.io/docs/connector-create).
+- [Create a Connector with Function worker](https://pulsar.apache.org/docs/io-quickstart/).
+Using this way requires you to download a **NAR** package to create a connector. You can download the version you need from the `download button` at the beginning of the article.
+- [Create a Connector with Function mesh](https://functionmesh.io/docs/connectors/run-connector).
+Using this way requires you to set the docker image. You can choose the version you want to launch from [here](https://hub.docker.com/r/streamnative/pulsar-io-kinesis).
+
+No matter how you create an AWS Kinesis sink connector, the minimum configuration contains the following parameters.
+```yaml
+configs:
+awsRegion: "Your aws kinesis region"
+awsKinesisStreamName: "Your kinesis stream name"
+awsCredentialPluginParam: "{\"accessKey\":\"Your access key\",\"secretKey\":\"Your secret key\"}"
+```
```

Comment:
How about we guide users here to get the awsRegion and awsCredentialPluginParam? We could just copy the description of `awsRegion` and `awsCredentialPluginParam` from the `Configuration Properties` chapter. Or we could link to them.

Reply:
1. We covered how to get `AccessKey` and `SecretKey ` in **Prerequisites**.
2. `awsRegion` is a very easy concept to understand, and we've covered it in the next line that users can go to **Configuration Properties** for a more detailed explanation
> If you want to configure more parameters, see [Configuration Properties](https://github.com/streamnative/pulsar-hub/diffs/0?base_sha=7e8c2208e934f8bab9ee68fa486b15f719ca2312&branch=improve_kinesis_sink&commentable=true&name=improve_kinesis_sink&pull_number=290&qualified_name=refs%2Fheads%2Fimprove_kinesis_sink&sha1=7e8c2208e934f8bab9ee68fa486b15f719ca2312&sha2=b24e8bfbb5ff06fcf8d17626911973aaca900b4d&short_path=eeb84b4&unchanged=expanded&w=false#configuration-properties) for reference.

Reply:
Make sense to me.